sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -522,6 +522,8 @@ case class JsonTuple(children: Seq[Expression])
     Examples:
       > SELECT _FUNC_('{"a":1, "b":0.8}', 'a INT, b DOUBLE');
        {"a":1,"b":0.8}
+      > SELECT from_json('{"teacher": "Alice", "student": [{"name": "Bob", "rank": 1}, {"name": "Charlie", "rank": 2}]}', 'STRUCT<teacher: STRING, student: ARRAY<STRUCT<name: STRING, rank: INT>>>');
```

Comment:
Shouldn't this be "SELECT _FUNC_" here for consistency? otherwise seems fine

Reply:
BTW, I would add this example in the end below `timestampFormat` example considering that this example is more complicated then others, and the examples should ideally show from easy ones to harder ones

Reply:
Thanks for your review! I've updated the code. Could you please help with the test failure?